docs/docs/en/guide/task/switch.md

```diff
@@ -14,26 +15,42 @@ Drag from the toolbar <img src="/img/switch.png" width="20"/>  task node to canv
 - Descriptive information: Describe the function of the node.
 - Task priority: When the number of worker threads is insufficient, execute in the order of priority from high to low, and tasks with the same priority will execute in a first-in first-out order.
 - Worker grouping: Assign tasks to the machines of the worker group to execute. If `Default` is selected, randomly select a worker machine for execution.
+- Environment name: The environment in Security, if not configured, it will not be used.
+- Task group name: The group in Resources, if not configured, it will not be used.
 - Times of failed retry attempts: The number of times the task failed to resubmit. You can select from drop-down or fill-in a number.
 - Failed retry interval: The time interval for resubmitting the task after a failed task. You can select from drop-down or fill-in a number.
+- Delay execution time: Task delay execution time.
 - Timeout alarm: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
 - Condition: You can configure multiple conditions for the switch task. When the conditions are satisfied, execute the configured branch. You can configure multiple different conditions to satisfy different businesses.
 - Branch flow: The default branch flow, when all the conditions are not satisfied, execute this branch flow.
 
-## Detail
+## Task Example
+
+This is demonstrated using one switch task and three shell tasks.
+
+![switch_01](/img/tasks/demo/switch_01.png)
+
+### 1. Create a workflow
```

Comment:
We should avoid using number in markdown title, the prefix `###` is for the same function @huangchenguang123  and cc @QuakeWang 
```suggestion
### Create a workflow
```

Reply:
same as others places